packages/contentstack-auth/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Contentstack",
   "bugs": "https://github.com/contentstack/cli/issues",
   "scripts": {
+    "pack": "pnpm clean && pnpm compile && oclif manifest && oclif readme",
     "build": "npm run clean && npm run compile",
```

Comment:
@antonyagustine npm pack is a built in command by npm for packaging the project, can you confirm using "pack" keyword for our scripts will effect the flow of publishing ? (npm publish internally uses npm pack command). Simple solution would be using a different keyword for it.

Reply:
@shafeeqd959 We changed the script name.